Core/clim-core/gadgets/concrete.lisp

```diff
@@ -1574,6 +1574,20 @@ if INVOKE-CALLBACK is given."))
   (popup-list-box pane)
   (disarm-gadget pane))
 
+(defmethod handle-event ((pane generic-option-pane) (event pointer-scroll-event))
+  (when (list-pane-exclusive-p pane)
+    (let ((delta-y (pointer-event-delta-y event))
+          (items (list-pane-items pane)))
+      (if (slot-boundp pane 'value)
+          (let* ((old-position (position (gadget-value pane) items
+                                         :test (list-pane-test pane)
+                                         :key (list-pane-value-key pane)))
+                 (new-position (alexandria:clamp (+ old-position (signum delta-y)) 0 (1- (length items)))))
+            (unless (= old-position new-position)
+              (setf (gadget-value pane :invoke-callback t) (nth new-position items))))
+          (when (and items (plusp delta-y)) ;; if no item is selected and we're scrolling down, select the first one.
```

Comment:
please put the comment above the form, or use single `;` for "after-line" comments